website/docs/r/web_application_firewall_policy.html.markdown

```diff
@@ -0,0 +1,149 @@
+---
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_web_application_firewall_policy"
+sidebar_current: "docs-azurerm-resource-web-application-firewall-policy"
+description: |-
+  Manage a Azure WebApplicationFirewallPolicy instance.
+---
+
+# azurerm_web_application_firewall_policy
+
+Manage a Azure WebApplicationFirewallPolicy instance.
+
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "example" {
+  name     = "example-rg"
+  location = "West US 2"
+}
+
+resource "azurerm_web_application_firewall_policy" "example" {
+  name                = "example-wafpolicy"
+  resource_group_name = "${azurerm_resource_group.example.name}"
+  location            = "${azurerm_resource_group.example.location}"
+
+  custom_rules {
+    name      = "Rule1"
+    priority  = 1
+    rule_type = "MatchRule"
+
+    match_conditions {
+      match_variables {
+        variable_name = "RemoteAddr"
+      }
+
+      operator          = "IPMatch"
+      negation_conditon = false
+      match_values      = ["192.168.1.0/24", "10.0.0.0/24"]
+    }
+
+    action = "Block"
+  }
+
+  custom_rules {
+    name      = "Rule2"
+    priority  = 2
+    rule_type = "MatchRule"
+
+    match_conditions {
+      match_variables {
+        variable_name = "RemoteAddr"
+      }
+
+      operator          = "IPMatch"
+      negation_conditon = false
+      match_values      = ["192.168.1.0/24"]
+    }
+
+    match_conditions {
+      match_variables {
+        variable_name = "RequestHeaders"
+        selector      = "UserAgent"
+      }
+
+      operator          = "Contains"
+      negation_conditon = false
+      match_values      = ["Windows"]
+    }
+
+    action = "Block"
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name of the policy. Changing this forces a new resource to be created.
+
+* `resource_group_name` - (Required) The name of the resource group. Changing this forces a new resource to be created.
+
+* `location` - (Optional) Resource location. Changing this forces a new resource to be created.
+
+* `custom_rules` - (Optional) One or more `custom_rule` block defined below.
+
+* `policy_settings` - (Optional) One `policy_setting` block defined below.
+
+* `tags` - (Optional) Resource tags. Changing this forces a new resource to be created.
```

Comment:
Changing a tag results in a new policy being created?

Reply:
@nexxai No... that was my bad. Thanks for nice catch. 😆